apps/console/src/features/applications/utils/sign-in-method-utils.ts

```diff
@@ -344,3 +418,23 @@ export type ConnectionsJITUPConflictWithMFAReturnValue = {
     conflicting: boolean;
     idpList: GenericAuthenticatorWithProvisioningConfigs[];
 };
+
+export type FederatedConflictWithSMSOTPArgs = {
+    
+    /**
+     * This parameter should only pass in the configured federated
+     * authenticators under a tenant.
+     */
+    federatedAuthenticators: GenericAuthenticatorInterface[];
+    /**
+     * All the steps in the authentication sequence. Callee must pass
+     * all the authentication options without skipping any.
+     */
+    steps: AuthenticationStepInterface[];
+    subjectStepId: number;
+};
+
+export type FederatedConflictWithSMSOTPReturnValue = {
+    conflicting: boolean;
+    idpList: GenericAuthenticatorInterface[];
+}
```

Comment:
Can we move these to the models folder?

Reply:
Resolves. via [43c7195](https://github.com/wso2/identity-apps/pull/3782/commits/43c71954ca90804c2445db3cf260b16ebcfa26c9)